### PR TITLE
Update READMEs badges and links

### PR DESCRIPTION
**Pull request by chunnienc**
BUG=NO_BUG

### CODE REVIEW

File: ai_edge_torch/generative/README.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ The user needs to implement the entire LLM Pipeline themselves, and call TFLite
 
 This approach provides users with the most control. For example, they can implement streaming, get more control over system memory or implement advanced features such as constrained grammar decoding, speculative decoding etc.
 
-A very simple text generation pipeline based on a decoder-only-transformer is provided [here](https://github.com/google-ai-edge/ai-edge-torch-archive/blob/main/ai_edge_torch/generative/examples/c%2B%2B/text_generator_main.cc) for reference. Note that this example serves as a starting point, and users are expected to implement their own pipelines based on their model's specific requirements.
+A very simple text generation pipeline based on a decoder-only-transformer is provided [here](https://github.com/google-ai-edge/ai-edge-torch/blob/853301630f2b2455bd2e2f73d8a47e1a1534c91c/ai_edge_torch/generative/examples/c%2B%2B/text_generator_main.cc) for reference. Note that this example serves as a starting point, and users are expected to implement their own pipelines based on their model's specific requirements.
```
Comment (hheydary) [inline ai_edge_torch/generative/README.md:81]: Can we keep this at main as opposed to a fixed commit?

File: ai_edge_torch/generative/README.md [hunk 2]
```diff
@@ -105,7 +105,7 @@ model-explorer 'gemma-2b.tflite'
 
 <img width="890" alt="Gemma-2b visualization demo" src="screenshots/gemma-tflite.png">
 
-For an end-to-end example showing how to author, convert, quantize and execute, please refer to the steps [here](https://github.com/google-ai-edge/ai-edge-torch-archive/blob/main/ai_edge_torch/generative/examples/README.md)
+For an end-to-end example showing how to author, convert, quantize and execute, please refer to the steps [here](https://github.com/google-ai-edge/ai-edge-torch/blob/853301630f2b2455bd2e2f73d8a47e1a1534c91c/ai_edge_torch/generative/examples/README.md)
```
Comment (hheydary) [inline ai_edge_torch/generative/README.md:108]: Same here.

**Review by hheydary — APPROVED** [review]

**Review by majiddadashi — DISMISSED** [review]

**Review by hheydary — APPROVED** [review]

File: ai_edge_torch/generative/examples/README.md [hunk 3]
```diff
@@ -22,10 +22,10 @@ For each of the example models, we have a model definition file (e.g. tiny_llama
 Here we use `TinyLlama` as an example to walk you through the authoring steps.
 
 #### Define model's structure
-https://github.com/google-ai-edge/ai-edge-torch-archive/blob/e54638dd4a91ec09115f9ded1bd5540f3f1a4e68/ai_edge_torch/generative/examples/tiny_llama/tiny_llama.py#L43-L74
+https://github.com/google-ai-edge/ai-edge-torch/blob/853301630f2b2455bd2e2f73d8a47e1a1534c91c/ai_edge_torch/generative/examples/tiny_llama/tiny_llama.py#L43-L74
```
Comment (haozha111) [inline ai_edge_torch/generative/examples/README.md:25]: https://github.com/google-ai-edge/ai-edge-torch/blob/853301630f2b2455bd2e2f73d8a47e1a1534c91c/ai_edge_torch/generative/examples/tiny_llama/tiny_llama.py#L46-L77

File: ai_edge_torch/generative/examples/README.md [hunk 4]
```diff
@@ -22,10 +22,10 @@ For each of the example models, we have a model definition file (e.g. tiny_llama
...
 
 #### Define model's forward function
-https://github.com/google-ai-edge/ai-edge-torch-archive/blob/e54638dd4a91ec09115f9ded1bd5540f3f1a4e68/ai_edge_torch/generative/examples/tiny_llama/tiny_llama.py#L79-L101
+https://github.com/google-ai-edge/ai-edge-torch/blob/853301630f2b2455bd2e2f73d8a47e1a1534c91c/ai_edge_torch/generative/examples/tiny_llama/tiny_llama.py#L79-L101
```
Comment (haozha111) [inline ai_edge_torch/generative/examples/README.md:28]: https://github.com/google-ai-edge/ai-edge-torch/blob/853301630f2b2455bd2e2f73d8a47e1a1534c91c/ai_edge_torch/generative/examples/tiny_llama/tiny_llama.py#L79-L104

File: ai_edge_torch/generative/examples/README.md [hunk 5]
```diff
@@ -37,12 +37,12 @@ place to simplify the `state_dict` mapping process (`utilities/loader.py`).
 The user needs to provide a layer name tempelate (TensorNames) for the source
 model. This tempelate is then used to create an updated `state_dict` that works
 with the mapped model. The tensor map includes the following fields:
-https://github.com/google-ai-edge/ai-edge-torch-archive/blob/3b753d80fdf00872baac523dc727b87b3dc271e7/ai_edge_torch/generative/utilities/loader.py#L120-L134
+https://github.com/google-ai-edge/ai-edge-torch/blob/853301630f2b2455bd2e2f73d8a47e1a1534c91c/ai_edge_torch/generative/utilities/loader.py#L120-L134
```
Comment (haozha111) [inline ai_edge_torch/generative/examples/README.md:40]: https://github.com/google-ai-edge/ai-edge-torch/blob/853301630f2b2455bd2e2f73d8a47e1a1534c91c/ai_edge_torch/generative/utilities/loader.py#L94-L109

File: ai_edge_torch/generative/examples/README.md [hunk 6]
```diff
@@ -37,12 +37,12 @@ place to simplify the `state_dict` mapping process (`utilities/loader.py`).
...
 
 The fields that have a default value of `None` are optional and should only be
 populated if they are relevant to the model architecture. For `TinyLlama`, we
 will define the following `TENSOR_NAMES`:
-https://github.com/google-ai-edge/ai-edge-torch-archive/blob/e54638dd4a91ec09115f9ded1bd5540f3f1a4e68/ai_edge_torch/generative/examples/tiny_llama/tiny_llama.py#L27-L40
+https://github.com/google-ai-edge/ai-edge-torch/blob/853301630f2b2455bd2e2f73d8a47e1a1534c91c/ai_edge_torch/generative/examples/tiny_llama/tiny_llama.py#L27-L40
```
Comment (haozha111) [inline ai_edge_torch/generative/examples/README.md:45]: https://github.com/google-ai-edge/ai-edge-torch/blob/853301630f2b2455bd2e2f73d8a47e1a1534c91c/ai_edge_torch/generative/examples/tiny_llama/tiny_llama.py#L30-L43

File: ai_edge_torch/generative/examples/README.md [hunk 7]
```diff
@@ -59,7 +59,7 @@ using a few input samples before proceeding to the conversion step.
 
 ### Model conversion
 In this step, we use the `ai_edge_torch`'s standard multi-signature conversion API to convert PyTorch `nn.Module` to a single TFLite flatbuffer for on-device execution. For example, in `tiny_llama/convert_to_tflite.py`, we use this python code to convert the `TinyLLama` model to a multi-signature TFLite model:
-https://github.com/google-ai-edge/ai-edge-torch-archive/blob/3b753d80fdf00872baac523dc727b87b3dc271e7/ai_edge_torch/generative/examples/tiny_llama/convert_to_tflite.py#L22-L53
+https://github.com/google-ai-edge/ai-edge-torch/blob/853301630f2b2455bd2e2f73d8a47e1a1534c91c/ai_edge_torch/generative/examples/tiny_llama/convert_to_tflite.py#L22-L53
```
Comment (haozha111) [inline ai_edge_torch/generative/examples/README.md:62]: https://github.com/google-ai-edge/ai-edge-torch/blob/853301630f2b2455bd2e2f73d8a47e1a1534c91c/ai_edge_torch/generative/examples/tiny_llama/convert_to_tflite.py#L26-L61

**Review by haozha111 — APPROVED** [review]

**Pull request merged**